modules/labs-react/combobox/lib/Combobox.tsx

```diff
@@ -409,12 +417,13 @@ const Combobox = ({
         paddingRight: space.xl,
       };
     }
-    const newTextInputProps: Partial<TextInputProps> = {
+    const newTextInputProps: Partial<TextInputProps &
+      React.InputHTMLAttributes<HTMLInputElement> & {ref: React.Ref<HTMLInputElement>}> = {
```

Comment:
I had to modify this type definition to get TS to stop complaining about the `ref` below. I don't think this definition is necessary (TS seems to be able to infer just fine without it), however, so I may remove it.